Spatial.tscn

```diff
@@ -217,51 +246,72 @@ __meta__ = {
 
 [node name="VBoxContainer" type="VBoxContainer" parent="Control/Dialogs/MainMenu/ScrollContainer"]
 margin_right = 220.0
-margin_bottom = 284.0
+margin_bottom = 368.0
 size_flags_horizontal = 3
 
 [node name="LoadPath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
 margin_right = 220.0
 margin_bottom = 40.0
 rect_min_size = Vector2( 0, 40 )
-text = "Open Markers File"
+text = "Toggle Marker Visibility"
 
-[node name="SavePath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
+[node name="HSeparator5" type="HSeparator" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
 margin_top = 44.0
 margin_right = 220.0
-margin_bottom = 84.0
+margin_bottom = 48.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="ImportPath" type="Button" parent="Control/Dialogs/MainMenu/ScrollContainer/VBoxContainer"]
+margin_top = 52.0
+margin_right = 220.0
+margin_bottom = 72.0
+text = "Import Marker Pack"
```

Comment:
This button is a different size from the other buttons.

Reply:
I missed adding the min size to it so it kept readjusting. Added.